docs/source/guidelines/security.rst

```diff
@@ -0,0 +1,41 @@
+Security
+########
+
+The OpenKAT project is committed to providing a secure environment.
+As a security product ourselves, we take security seriously and strive to be transparent about our security posture.
+
+Security in the pipeline
+=========================
+
+The OpenKAT project uses a variety of tools to ensure the security of the coWdebase. These include:
+
+SCA
+---
+
+To scan for vulnerabilities in our code base we use the built-in tool from Github, Dependabot.
+This tool scans the code base for outdated components and vulnerabilities, and creates a pull request to update the dependencies.
+This way we can ensure that the code base is up to date and secure.
+
+SAST
+----
+
+To scan for code smells in our code base we use the built-in tool from Github, CodeQL.
```

Comment:
I think *code smell* is not the correct term for this. Code smells are traces in the code that indicate a deeper problem that should be addressed or refactored. They are not necessarily errors or bugs but rather violations of design principles and best practices (aka bad code) which can lead to maintainability issues.

CodeQL focuses on discovering *vulnerabilities* in the codebase, whereas linters are more commonly used to detect *code smells*.

(You *could* use CodeQL to detect code smells by querying AST trees to find specific patterns, but this requires writing intricate queries and is not the primary focus of CodeQL.)